### PR TITLE
4.x: Fix XML document spelling errors

### PR DESCRIPTION
**Pull request by akarnokd**
This PR fixes a bunch of spelling errors in the XML documentation of the public API and then some.

I have left alone the `cancellable` and similar worlds because of the US/UK spelling style differences (and somehow the double L comes more natural to me despite the rules).

### CODE REVIEW

File: Rx.NET/Source/src/System.Reactive/System.Reactive.csproj [hunk 1]
```diff
@@ -53,8 +53,22 @@
   <ItemGroup>
     <None Update="Linq\Observable\CombineLatest.Generated.tt" Generator="TextTemplatingFileGenerator" LastGenOutput="CombineLatest.Generated.cs" />
     <Compile Update="Linq\Observable\CombineLatest.Generated.cs" DesignTime="True" AutoGen="True" DependentUpon="CombineLatest.Generated.tt" />
+    <Compile Update="Strings_Linq.Designer.cs">
```
Comment (akarnokd) [inline Rx.NET/Source/src/System.Reactive/System.Reactive.csproj]: Not sure how this ended up here.

Comment (danielcweber) [reply Rx.NET/Source/src/System.Reactive/System.Reactive.csproj]: Probably due to changed master. Rebase, I'll merge.

Comment (akarnokd) [reply Rx.NET/Source/src/System.Reactive/System.Reactive.csproj]: I started fresh with this, the update button is greyed out. I'll undo this change.

**Comment by akarnokd** [comment]
Maybe because I touched those resx files, VS thought why not mess up the project settings with all sorts of different tool versions.

**Comment by danielcweber** [comment]
Is it good to merge then?

**Comment by akarnokd** [comment]
it passed, so yes.

File: Rx.NET/Source/src/System.Reactive/Strings_Core.Designer.cs [hunk 2]
```diff
@@ -71,7 +71,7 @@ internal class Strings_Core {
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to OnCompleted notification doesn&apos;t have a value..
+        ///   Looks up a localized string similar to OnCompleted notification doesn't have a value..
```
Comment (danielcweber) [inline Rx.NET/Source/src/System.Reactive/Strings_Core.Designer.cs]: I guess this typo is by design and will be regenerated.

**Pull request merged**